plugins/privacy/actionlogs/actionlogs.php

```diff
@@ -36,12 +37,14 @@ public function onPrivacyExportRequest(PrivacyTableRequest $request, JUser $user
 		}
 
 		$domain = $this->createDomain('user_action_logs', 'joomla_user_action_logs_data');
+		$db     = $this->db;
 
-		$query = $this->db->getQuery(true)
-			->select('a.*, u.name')
-			->from('#__action_logs AS a')
-			->innerJoin('#__users AS u ON a.user_id = u.id')
-			->where($this->db->quoteName('a.user_id') . ' = ' . $user->id);
+		$query = $db->getQuery(true)
+			->select($db->quoteName(['a.*', 'u.name']))
+			->from($db->quoteName('#__action_logs', 'a'))
+			->innerJoin($db->quoteName('#__users', 'u'), $db->quoteName('a.user_id') . ' = ' . $db->quoteName('u.id'))
+			->where($this->db->quoteName('a.user_id') . ' = :id')
```

Comment:
For consistency here.
```suggestion
			->where($db->quoteName('a.user_id') . ' = :id')
```